arkouda/series.py

```diff
@@ -149,33 +154,29 @@ def __repr__(self):
             prt = pd.concat(
                 [self.head(maxrows // 2 + 2).to_pandas(), self.tail(maxrows // 2).to_pandas()]
             )
-            length_str = "\nLength {len(self)}"
+            length_str = f"\nLength {len(self)}"
         return (
             prt.to_string(
-                dtype=prt.dtype,
-                min_rows=get_option("display.min_rows"),
-                max_rows=maxrows,
-                length=False,
+                dtype=prt.dtype, min_rows=get_option("display.min_rows"), max_rows=maxrows, length=False,
             )
             + length_str
         )
 
     def __getitem__(self, key):
-        if type(key) == Series:
+        if isinstance(key, Series):
             # @TODO align the series indexes
             key = key.values
         return Series((self.index[key], self.values[key]))
 
     dt = CachedAccessor("dt", DatetimeAccessor)
-    str = CachedAccessor("str", StringAccessor)
-    # cat = CachedAccessor("cat", CategoricalAccessor)
+    str_acc = CachedAccessor("str", StringAccessor)
 
     @property
     def shape(self):
         # mimic the pandas return of series shape property
         return (self.values.size,)
 
-    def isin(self, lst):
+    def isin(self, lst: Union[pdarray, Strings]) -> Series:
```

Comment:
i think you missed the `@typechecked` above `isin` here

Reply:
Done